### PR TITLE
check ownership when leftclick and add infotext to node of who is the…

### PR DESCRIPTION
**Pull request by berengma**
in last commit I forgot to check owner when leftclick the anvil
this is fixed now.

also I added owner showing on anvil's infotext

cleaned up code and removed --debug

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -221,107 +222,109 @@ minetest.register_node("anvil:anvil", {
...
-				or input:get_name() == "technic:water_can" 
-				or input:get_name() == "technic:lava_can" ) then
-			return
-		end
-	
-		-- 65535 is max damage
-		local damage_state = 40-math.floor(input:get_wear()/1638)
-	
-		local tool_name = input:get_name()
+		if meta:get_string("owner") == puncher:get_player_name() then
```
Comment (SmallJoker) [inline init.lua]:
```Lua
if meta:get_string("owner") ~= puncher:get_player_name() then
	return
end
```
Less indents, same effect, better readability.

Comment (berengma) [reply init.lua]: thanks, I think I know now what you mean

**Comment by SmallJoker** [comment]
Thank you for the change, although the indents are now inconsistent. Spaces and tabs are mixed, whereas there should only be tabs (coding style of this mod).

**Comment by berengma** [comment]
@SmallJoker 
I can not explain the spaces. I only use tab key in my editor (kate in kde).
Maybe I'll try with vi tomorrow.

**Comment by berengma** [comment]
I think spaces are fixed now.

**Comment by SmallJoker** [comment]
e91e913

**Pull request closed without merging**